Makefile

```diff
@@ -14,6 +14,8 @@ help:
 	@echo "        Apply black formatting to code."
 	@echo "    lint"
 	@echo "        Lint code with flake8, and check if black formatter should be applied."
```

Comment:
what do you think about including this as part of `make lint`?

Reply:
`make lint` runs against the entire codebase and not just the diff. I could concatenate it with `&&` though 🤔

Reply:
done